app/main.py

```diff
@@ -1 +1,48 @@
-#  write your code here
+from typing import List
+
+
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int = 0,
```

Comment:
You should remove default value from `appetite`, there no animal that doesn't eat